src/ecalc/libraries/libecalc/common/tests/input/validation/snapshots/test_validation_json_schemas/test_json_schema_changed/schemas.json

```diff
@@ -1266,6 +1391,7 @@
                 }
             },
             "required": [
+                "FUEL_TYPES",
```

Comment:
Is it required?

Reply:
yes, fuel for a consumer is required, so therefore this is required as we only support a reference